Okta.AspNet.Abstractions/OktaWebOptions.cs

```diff
@@ -36,5 +36,10 @@ public class OktaWebOptions
         /// The clock skew.
         /// </value>
         public TimeSpan ClockSkew { get; set; } = DefaultClockSkew;
+
+        /// <summary>
+        /// Gets or sets the uri of your organization's proxy server.  The default is null.
```

Comment:
Nit: 
Let's use URI instead of uri. 
We can also use [the tag `<c>` or `<code>`](https://docs.microsoft.com/en-us/dotnet/csharp/codedoc#c) for `null`.